src/pycliarr/api/sonarr.py

```diff
@@ -336,4 +335,23 @@ def get_queue(
         """
         return self._get_queue(
             page, sort_key, page_size, sort_dir, options={"includeUnknownSeriesItems": include_unknown}
```

Comment:
you moved the parenthesis, so this is not closed anymore, it won't work

Reply:
Fixed with your patch, thanks!